xrpl/models/transactions/xchain_claim.py

```diff
@@ -81,13 +80,13 @@ def _get_errors(self: Self) -> Dict[str, str]:
 
         amount = self.amount
         if is_xrp(amount):
-            currency = XRP()
+            currency: Currency = XRP()
         elif is_issued_currency(amount):
-            currency = amount.to_currency()  # type: ignore
+            currency = amount.to_currency()
         elif is_mpt(amount):
-            currency = amount.mpt_issuance_id  # type: ignore
+            currency = amount.to_currency()
```

Comment:
both these conditions can be combined into one line

Reply:
IMO it's clearer this way and you don't really get any performance benefit from combining it